projects/igniteui-angular/src/lib/grids/grid/grid-filtering.spec.ts

```diff
@@ -649,6 +649,19 @@ describe('IgxGrid - Filtering actions #grid', () => {
         expect(grid.rowList.length).toEqual(1);
     }));
 
+    it('should exclude null and undefined values when filter by \'false\' and column dataType is not set', fakeAsync(() => {
+        grid.columns[3].dataType = undefined;
+        grid.filterMode = 'excelStyleFilter';
+        fix.detectChanges();
```

Comment:
These steps are necessary when creating the filter in the UI, since the test is using the `filter` method directly, they can be removed.